### PR TITLE
feat(env): allow default values with ${ENV_VAR:-defaultValue}

### PR DESCRIPTION
**Pull request by davezuko**

### CODE REVIEW

**Comment by Fluidbyte** [comment]
Looks good, could you just add some docs - I like to add stuff like this to the example at the top of the README so it's obvious, then a one-liner in the body of the docs to cover details.

**Comment by Fluidbyte** [comment]
Clinton Approves ...after docs...

**Comment by davezuko** [comment]
@Fluidbyte added docs here: https://github.com/binci/binci/pull/85/files#diff-04c6e90faac2675aa89e2176d2eec7d8R214.

File: README.md [hunk 1]
```diff
@@ -209,7 +209,9 @@ binci --cleanup-all
 
 ## Environment Variables (`env <array>`)
 
-Setting `env` array items will expose environment variables in the primary instance or services. These entries can be raw strings or use `${VAR}` notation, where `VAR` is an environment variable on the host machine to use. Entries should use the format `<ENV_VAR>=<VALUE>`
+Setting `env` array items will expose environment variables in the primary instance or services. These entries can be raw strings or use `${VAR}` notation, where `VAR` is an environment variable on the host machine to use. Entries should use the format `<ENV_VAR>=<VALUE>`.
```
Comment (Fluidbyte) [inline README.md]: Looks good 👍 , Could you add an implementation example in the `env` block on L87?

Comment (davezuko) [reply README.md]: Done!

Comment (Fluidbyte) [reply README.md]: you rock

**Pull request merged**